flowmachine/flowmachine/core/mixins/geodata_mixin.py

```diff
@@ -243,16 +244,32 @@ def _get_geojson(self, proj4):
         dict
 
         """
-        features = [
-            x[0] for x in self.connection.fetch(self._geo_json_query(crs=proj4))
-        ]
+        features = [x[0] for x in self.connection.fetch(self.geojson_query(crs=proj4))]
         js = {
             "properties": {"crs": proj4},
             "type": "FeatureCollection",
             "features": features,
         }
         return js
 
+    def get_proj4_string(self, crs=None):
```

Comment:
Not immediately clear why this is necessary, given it just wraps the util method?

Reply:
It was necessary to get the proj4 string in `server.py`, so that it could be returned to FlowAPI to include in the geojson. But since that shouldn't be there, this isn't necessary any more.